c-cannons/bangazon-initial_site
# Hc add to cart

## Pull Request
#### Descriptions

1. App checks for existence of open order upon login
1. If open order exists, that order serves as the shopping cart
1. If no open order exists, the app creates one
1. User can add items to the shopping cart.
1. User can clear all items from the shopping cart.
1. Fixed error in Order History caused by absence of payment method in open orders

#### Steps to Test

You must provide clear steps for any teammate to test the code.

1. `git fetch origin HC-add-to-cart`
1. `git checkout HC-add-to-cart`
1. Navigate to the shopping cart to see if any items exist
1. If items are in cart, click "Clear Shopping Cart" to remove items
1. If no items are in cart, or user clears cart, navigate to a product via a search or the category menu and click "Add to Cart"
1. The user should be able to add items from search or category listings, or from the "More Info" product card
1. Navigate to the cart to confirm that the correct items appear in the cart

#### Link to Feature Ticket

[#9](https://github.com/c-cannons/bangazon-initial_site/issues/9)
